web/src/games/chess/ai.ts

```diff
@@ -17,7 +17,7 @@ class Stockfish {
   worker: any;
 
   constructor() {
-    this.worker = new StockfishWorker();
```

Comment:
Is it possible to create a d.ts file containing a simple definition of `StockfishWorker` rather than casting to `any`?

Reply:
I'll need to look into this - I'd imagine it would be, but I don't know how quite yet.